src/pynxtools/dataconverter/validation.py

```diff
@@ -476,6 +476,20 @@ def handle_attribute(node: NexusNode, keys: Mapping[str, Any], prev_path: str):
                 f"{prev_path}/{variant if variant.startswith('@') else f'@{variant}'}",
             )
 
+            # Check enumeration
+            if (
+                node.items is not None
+                and mapping[
+                    f"{prev_path}/{variant if variant.startswith('@') else f'@{variant}'}"
+                ]
+                not in node.items
+            ):
+                collector.collect_and_log(
+                    f"{prev_path}/{variant if variant.startswith('@') else f'@{variant}'}",
+                    ValidationProblem.InvalidEnum,
+                    node.items,
+                )
+
```

Comment:
This does not quite work yet. If there are no variants, it does not check the enum rn.

Reply:
The same is actually apparently true for fields... (I copied the code from there).

Reply:
I added a fix for this, should work correctly now. Tests would still be very good to add.

Reply:
Thanks for the fix. I'll check it out. I'll add a test as well tomorrow.